artifactory/spec/specfiles.go

```diff
@@ -111,15 +111,20 @@ func (f File) IsSymlinks(defaultValue bool) (bool, error) {
 	return clientutils.StringToBool(f.Symlinks, defaultValue)
 }
 
-func (f *File) ToArtifactoryCommonParams() *utils.ArtifactoryCommonParams {
+func (f *File) ToArtifactoryCommonParams() (*utils.ArtifactoryCommonParams, error) {
+	var err error
```

Comment:
It appears that this err declaration is redundant. The err can be defined using := when returned from the ```utils.ParseProperties``` function.